### PR TITLE
Shailesh/unit system

### PR DESCRIPTION
**Pull request by Shailesh-05**

### CODE REVIEW

File: apps/weight-converter/src/App.jsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const ColContainer = styled.div`
 // `data` prop is exactly what is returned by queryToData.
 
 function Component({ data }) {
+
```
Comment (sahil-shubham) [inline apps/weight-converter/src/App.jsx:25]: This seems unnecessary, let's remove this change.

File: apps/unit-system/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+import { Component, queryToData } from "./App";
+
+const App: InstantApp = {
+  id: "@felvin-search-apps/unit-system",
+  name: "Converting given length to metre(SI unit)",
+  description: "This is a app which gives you the measurement in S.I unit metre from any kind of unit.",
+  queryToData,
+  Component,
+  screenshotPath: "./files/screenshot.png",
+  exampleSearchQueries: ["convert 32 inch to metres"],
```
Comment (sahil-shubham) [inline apps/unit-system/src/index.ts:11]: The app doesn't work for this example. Seems like a typo.

```suggestion
  exampleSearchQueries: ["convert 32 inch to metre"],
```

File: apps/unit-system/src/index.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import { InstantApp } from "@felvin-search/core";
+import { Component, queryToData } from "./App";
+
+const App: InstantApp = {
+  id: "@felvin-search-apps/unit-system",
+  name: "Converting given length to metre(SI unit)",
+  description: "This is a app which gives you the measurement in S.I unit metre from any kind of unit.",
+  queryToData,
+  Component,
+  screenshotPath: "./files/screenshot.png",
```
Comment (sahil-shubham) [inline apps/unit-system/src/index.ts:10]: There doesn't seem to a be a screenshot at this path. Let's make a `files` folder inside `src` and add a `screenshot.png`

**Review by sahil-shubham — APPROVED** [review]
This works very well, I am amazed by how quickly you are picking things up considering this is your first time working with React. 

I suggested a few changes, give them a look. We can merge this pull request after them.

File: apps/weight-converter/src/App.jsx (hunk 1 above)
Comment (Shailesh-05) [reply apps/weight-converter/src/App.jsx:25]: Sure Sahil, There seems to be minor changes to be done here and there. Will finish those, add commits and then update you. Thanks for pointing out!

**Comment by sahil-shubham** [comment]
I made some changes because you seemed to have copied the screenshot of the weight convertor rather than adding a new screenshot and some other thing. Review what files you are pushing or committing carefully before doing so. 

Thanks for the contribution, merging this now.

**Pull request merged**